server/test/unit/server/webservices/test_urls.py

```diff
@@ -4,6 +4,9 @@
 
 
 class TestDjangoPluginsUrls(unittest.TestCase):
```

Comment:
@bmbouter, the advantage of breaking it up is that the tests are a little more organized and the individual tests do not need to be namespaced as carefully. I did them this way since so many PRs will go in side by side to prevent accidental clash between them. Also, since these are the only tests in this file, they will be contained together.

That being said, those reasons are minor and it does make sense to test them together since the urls are all contained in a single class.